### PR TITLE
Readme file

### PR DESCRIPTION
**Pull request by vikky-10**
check it tile now it is 🆗

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-  #  Tic-Tac-Toe-Game-Python
+ # 🎮 Tic-Tac-Toe-Game-Python
 
 We created a tic tac toe game using python
```
Comment (anilfth) [inline README.md]: we are going to

File: README.md [hunk 2]
```diff
@@ -85,10 +88,10 @@ Step: 2
 
 ## 🚀 Tech use
 
-Python
-tkinter
-Different lib.
+    Python
+    tkinter
+    Different lib.
```
Comment (anilfth) [inline README.md]: remove this

Comment (vikky-10) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (vikky-10) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (vikky-10) [reply README.md]: plz check

**Comment by anilfth** [comment]
good work @vikky-10

**Pull request merged**